cfme/fixtures/v2v_fixtures.py

```diff
@@ -41,19 +42,24 @@ def set_skip_event_history_flag(appliance):
         appliance.wait_for_web_ui()
 
 
+def _is_osp_worker_running(appliance):
+    return "started" in appliance.ssh_client.run_rake_command(
+        "evm:status | grep 'Openstack::Cloud::EventCatcher'"
+    ).output
+
+
 def _start_event_workers_for_osp(appliance, provider):
     """This is a workaround to start event catchers until BZ 1753364 is fixed"""
     provider_edit_view = navigate_to(provider, 'Edit')
     endpoint_view = provider.endpoints_form(parent=provider_edit_view)
     endpoint_view.events.event_stream.select("AMQP")
     endpoint_view.events.event_stream.select("Ceilometer")
     provider_edit_view.save.click()
-    # Test if workers are enabled and running
```

Comment:
Suggestion : Move this method in here . 
     
      
     def _is_osp_worker_running():
         return "started" in appliance.ssh_client.run_rake_command( "evm:status | grep  'Openstack::Cloud::EventCatcher'").output
     try:
           wait_for(lambda: _is_osp_worker_running(), num_sec=60, delay=10)
     except TimedOutError:
            pytest.skip("Openstack Eventcatcher workers are not running")